### ISSUE TITLE
Moveable QB Slots save incompatibility issues

### ISSUE THREAD

**Issue by CodeBatcher** [NewGenerationAlms/NgaShareableCode#5]
Moveable QB slots doesn't save the position nor the rotation of the already adjusted Quick belt highlights.

**Comment by NewGenerationAlms** [comment]
Same reason as the other comment, this mod's not supported any longer. However, I recall from the demo video that vault saving did work, not sure why it doesn't any longer. Some of my code had dependencies on CityRobo's code, so there's a small chance their code might've changed and now it's broken?

_These last mods I've been publishing have less user-instructions as I have less time to dedicate to modding and they're mainly to augment my gameplay to my personal preferences. I post the code here so others can copy it and make spinoffs if they like :)_

**Comment by CodeBatcher** [comment]
Yeah that seems to be the case at the moment. It doesn't save the slots position and rotation.

________________________________
